example_client/README.md

```diff
@@ -1,13 +1,104 @@
-# gRPC client examples
+# OpenVINO™ Model Server Example Client 
 
-This client can be used to demonstrate connectivity with OpenVINO Model Server over gRPC API and TensorFlow server using Predict call.
+This document contains examples to run *GetModelStatus*, *GetModelMetadata*, *Predict* functions over gRPC API and REST API.
 
+It covers following topics:
+* <a href="#grpc-api">gRPC API Client Examples </a>
+* <a href="#rest-api">REST API Client Examples  </a>
 
-## Requirements
+## Requirement
+
+Install client dependencies using the command :
+```
+pip3 install -r client_requirements.txt
+```
+
```

Comment:
For all of this examples we should specify cmd for starting OMVS and downloading proper models